### PR TITLE
Migrate Adding Inline Token test to Playwright

### PR DESCRIPTION
**Pull request by pooja-muchandikar**
## What?

Part of https://github.com/WordPress/gutenberg/issues/38851.
Migrate[ adding-inline-tokens.test.js ](https://github.com/WordPress/gutenberg/blob/trunk/packages/e2e-tests/specs/editor/various/adding-inline-tokens.test.js) to its Playwright version.
## Why?

Part of https://github.com/WordPress/gutenberg/issues/38851.
## How?

See[ MIGRATION.md](https://github.com/WordPress/gutenberg/blob/trunk/test/e2e/MIGRATION.md) for migration steps.
## Testing Instructions

Run` npm run test:e2e:playwright test/e2e/specs/editor/various/adding-inline-tokens.spec.js`

### CODE REVIEW

**Comment by pooja-muchandikar** [comment]
Hi @kevin940726,

I hope you are going great!..

Could you please help me in reviewing this PR?

Thanks!!

File: test/e2e/specs/editor/various/adding-inline-tokens.spec.js [hunk 1]
```diff
@@ -47,26 +40,21 @@ describe( 'adding inline tokens', () => {
...
 		// Insert the uploaded image.
-		await page.click( '.media-modal button.media-button-select' );
+		await page.click( 'role=button[name="Select"i]' );
 
 		// Check the content.
 		const regex = new RegExp(
 			`<!-- wp:paragraph -->\\s*<p>a <img class="wp-image-\\d+" style="width:\\s*10px;?" src="[^"]+\\/${ filename }\\.png" alt=""\\/?><\\/p>\\s*<!-- \\/wp:paragraph -->`
 		);
-		expect( await getEditedPostContent() ).toMatch( regex );
+		expect( await editor.getEditedPostContent() ).toMatch( regex );
```
Comment (kevin940726) [inline test/e2e/specs/editor/various/adding-inline-tokens.spec.js]: Let's instead do something like:

```js
const contentRegex = new RegExp(
	`a <img class="wp-image-\\d+" style="width:\\s*10px;?" src="[^"]+\\/${ filename }\\.png" alt=""\\/?>`
);
await expect.poll( editor.getBlocks ).toMatchObject( [ {
	name: 'core/paragraph',
	attributes: { content: expect.stringMatching( contentRegex ) }
} ] );
```

File: test/e2e/specs/editor/various/adding-inline-tokens.spec.js [hunk 2]
```diff
@@ -76,6 +64,6 @@ describe( 'adding inline tokens', () => {
 		const regex2 = new RegExp(
 			`<!-- wp:paragraph -->\\s*<p>a <img class="wp-image-\\d+" style="width:\\s*20px;?" src="[^"]+\\/${ filename }\\.png" alt=""\\/?><\\/p>\\s*<!-- \\/wp:paragraph -->`
 		);
-		expect( await getEditedPostContent() ).toMatch( regex2 );
+		expect( await editor.getEditedPostContent() ).toMatch( regex2 );
```
Comment (kevin940726) [inline test/e2e/specs/editor/various/adding-inline-tokens.spec.js]: Same here with a similar change.

**Review by kevin940726 — COMMENTED** [review]
Thank you! Sorry for the late review! This is looking great though! Just a small suggestion 🙇‍♂️.

**Comment by pooja-muchandikar** [comment]
Hi @kevin940726, 

I hope you are doing great!

I have addressed all the feedbacks and pushed the code, but seems PR has some conflicts however, I am not able to see these conflicts in my local. Could you please help in resolve it from PR directly or should I create new PR with the suggested changes?

**Comment by pooja-muchandikar** [comment]
Hi @kevin940726,
I am closing this PR as the conflicts are not visible in my local because of which unable to fix it.. 

So I have created new PR with all the changes/feedbacks being addressed - https://github.com/WordPress/gutenberg/pull/52020

Please review and let me know if there are more suggestions!

Thanks!

**Pull request closed without merging**